instrumentation/kafka/kafka-streams-0.11/javaagent/src/test/java/io/opentelemetry/javaagent/instrumentation/kafkastreams/KafkaStreamReflectionUtil.java

```diff
@@ -0,0 +1,112 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.javaagent.instrumentation.kafkastreams;
+
+import java.lang.reflect.Constructor;
+import java.lang.reflect.InvocationTargetException;
+import java.lang.reflect.Method;
+import java.util.Properties;
+import org.apache.kafka.common.serialization.Serde;
+import org.apache.kafka.common.serialization.Serdes;
+import org.apache.kafka.streams.KafkaStreams;
+import org.apache.kafka.streams.kstream.KStream;
+
+/**
+ * kafka stream reflection util which is used to compatible with different versions of kafka stream
+ */
+class KafkaStreamReflectionUtil {
+
+  private KafkaStreamReflectionUtil() {}
+
+  static Object createBuilder() throws Exception {
+    try {
+      // Different class names for test and latestDepTest.
+      return Class.forName("org.apache.kafka.streams.kstream.KStreamBuilder")
+          .getConstructors()[0]
```

Comment:
If you are looking for the no-arg constructor you should call `getConstructor()`. If the class has multiple constructors the order of elements in the array returned by `getConstructors` is unspecified.